redbot/cogs/mod/mod.py

```diff
@@ -58,6 +58,7 @@ class Mod(
         "current_tempbans": [],
         "dm_on_kickban": False,
         "default_days": 0,
+        "default_temban_duration": 1,
```

Comment:
Typo and default value needs update due to new format:
```suggestion
        "default_tempban_duration": 60 * 60 * 24,
```